### ISSUE TITLE
No Source Code has been published

### ISSUE THREAD

**Issue by grote** [David-Development/Task-Sync#3]
There's already a non-free CalDAV Task-Sync app, so there's no point in having another one. Please publsh the source coder under a free license, so we have at least one free app!

Thanks!

**Comment by bor** [comment]
It would be great!

**Comment by David-Development** [comment]
Publishing the source code hasn't been planned yet. Furthermore I want to earn some little money for my studies with this app.

**Comment by Bugsbane** [comment]
Fair enough. I understand needing to be able to eat / pay bills. That said I prefer to pay bounties on open source bugs / features than on keeping apps closed. As a matter of fact, I just offered to put money towards exactly that over on a competing (open source) project, Mirakel (also here on Github, so you can confirm I'm telling the truth).
